<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>march_Dialog</class>
 <widget class="QDialog" name="march_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редагування маршруту</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,6,1">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Додавання нового маршруту</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Номенр маршруту</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_5">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Створити</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="horizontalGroupBox">
       <property name="title">
        <string>Редактування маршруту</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="4,1">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Дані маршуру</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="3" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Індефікатор борту</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QPushButton" name="pushButton_3">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Примінити</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QComboBox" name="comboBox_2">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Номер маршруту</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Точка кінця</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QLineEdit" name="manevr_name_edit"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Точка початку</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QComboBox" name="comboBox_3">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Назва маршруту</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="route_number_edit">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="lineEdit_4"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Операції з маневрами</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Номер маневру</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLineEdit" name="lineEdit_5">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QPushButton" name="pushButton_4">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Додати</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QComboBox" name="comboBox_4">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="accessibleName">
                <string>Додати спеціальний</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_8">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Вибір маневру</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="4">
              <widget class="QComboBox" name="comboBox_5">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QPushButton" name="pushButton_7">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Видалити</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0" colspan="2">
              <widget class="QPushButton" name="pushButton_6">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Редагувати</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Маневри маршруту</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="6,1">
             <item>
              <widget class="QListWidget" name="listWidget"/>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QPushButton" name="up_button">
                 <property name="text">
                  <string>Вгору</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="down_button">
                 <property name="text">
                  <string>Вниз</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="add_Button">
                 <property name="text">
                  <string>Додати</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="delete_Button">
                 <property name="text">
                  <string>Видалити</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="edit_button">
                 <property name="text">
                  <string>Редагувати</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>march_Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>march_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
